### PR TITLE
write a script with conditions

### PR DESCRIPTION
**Pull request by AnastasiaMamontova**

### CODE REVIEW

File: Tasks/3 - Condition/condition.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <script>
+  var answer = prompt("What do you want to eat?");
+  if (answer === "seed") {
+  alert("Nice choice!");
+} else if (answer === "bread") {
```
Comment (gologames) [inline Tasks/3 - Condition/condition.html]: Please, align code. Copy and paste code [here](https://codebeautify.org/jsviewer), for example

Comment (gologames) [reply Tasks/3 - Condition/condition.html]: Ok, you have aligned JS code itself, but now you should align to the right all JS code up to the `<script>` tag

Like this for example:
image

File: Tasks/3 - Condition/condition.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <script>
+      var answer = prompt("What do you want to eat?");
```
Comment (gologames) [inline Tasks/3 - Condition/condition.html]: You can also use `const` for this variable

Comment (AnastasiaMamontova) [reply Tasks/3 - Condition/condition.html]: done

**Review by gologames — APPROVED** [review]
Great! You can merge PR now

**Pull request merged**